Svc/CmdSequencer/test/int/test_cmd_sequencer.py

```diff
@@ -0,0 +1,153 @@
+""" test_cmd_sequencer.py:
+
+A set of integration tests to apply to the CmdSequencer app. This is intended to be a reference of integration testing.
+"""
+
+import subprocess
+import time
+import os
+#from enum import Enum
+from pathlib import Path
+
+#from fprime_gds.common.testing_fw import predicates
+#from fprime_gds.common.utils.event_severity import EventSeverity
+
+## regenerate .seq file
+
+def regenerate_file(seqfile1,deployment):
+    # ex: test_seq.seq
+    oldfile1 = open (seqfile1 +".seq", "r")
+    newfile1 = open (seqfile1 +"_new.seq", "w")
+
+    eachline1 = oldfile1.readlines()
+    line = 1
+    for writing in eachline1:
+        
+        index = writing.find("cmdDisp.")
+        if index != -1:
+            writing = writing[:index] + deployment + "." + writing[index:]
+        
+        newfile1.write(writing)        
+        line = line + 1
+
+    oldfile1.close()
+    newfile1.close()
+    return seqfile1 +"_new.seq"
+
+def test_seqgen(fprime_test_api):
+    
+    """Tests the seqgen can be dispatched (requires localhost testing)"""
+
+    deployment =  fprime_test_api.get_deployment()
+    
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.CmdSequencer') + '.' + 'CS_AUTO', max_delay=1)
+    
+#    sequence = Path(__file__).parent / "test_seq_new.seq"
+#    sequence2 = Path(__file__).parent / "test_seq_wait_new.seq"
+
+    sequence = Path(__file__).parent / regenerate_file("test_seq",deployment)
```

Comment:
Why do you need to regenerate the sequence file? I think we can just use the original seq file.